tools/ci-build.sh

```diff
@@ -107,6 +107,12 @@ apt-get install -y --no-install-recommends \
   ${P4C_EBPF_DEPS} \
   ${P4C_RUNTIME_DEPS}
 
+# For Ubuntu 18.04, remove the Debian package and install the pypi-supplied version instead.
+if [[ "${DISTRIB_RELEASE}" == "18.04" ]] ; then
+apt remove --purge -y cmake
+pip3 install cmake==3.16.3
+fi
```

Comment:
By default the Ubuntu 18.04 container image does not have `cmake` installed. It might be better to remove `cmake` from `P4C_DEPS` above and use "if / else" statement like:
```bash
if [[ "${DISTRIB_RELEASE}" == "18.04" ]] ; then
    # For Ubuntu 18.04, remove the Debian package and install the pypi-supplied version instead.
    P4C_PIP_PACKAGES+=" cmake==3.16.3"
else
    P4C_DEPS+=" cmake"
fi
```